package.json

```diff
@@ -27,55 +27,54 @@
     "*": "prettier --ignore-unknown --write"
   },
   "dependencies": {
-    "@nestjs/common": "9.0.0",
-    "@nestjs/config": "2.3.3",
-    "@nestjs/core": "9.0.0",
-    "@nestjs/jwt": "10.0.3",
-    "@nestjs/passport": "9.0.3",
-    "@nestjs/platform-express": "9.0.0",
-    "@nestjs/swagger": "7.0.2",
-    "@nestjs/typeorm": "9.0.1",
-    "@prisma/client": "4.15.0",
+    "@nestjs/common": "10.0.5",
+    "@nestjs/config": "3.0.0",
+    "@nestjs/core": "10.0.5",
+    "@nestjs/jwt": "10.1.0",
+    "@nestjs/passport": "10.0.0",
+    "@nestjs/platform-express": "10.0.5",
+    "@nestjs/swagger": "7.1.1",
+    "@nestjs/typeorm": "10.0.0",
+    "@prisma/client": "4.16.2",
     "bcrypt": "5.1.0",
     "class-transformer": "0.5.1",
     "class-validator": "0.14.0",
-    "dotenv": "16.1.4",
+    "dotenv": "16.3.1",
     "passport": "0.6.0",
     "passport-jwt": "4.0.1",
     "passport-local": "1.0.0",
-    "pg": "8.11.0",
+    "pg": "8.11.1",
     "reflect-metadata": "0.1.13",
-    "rxjs": "7.2.0",
-    "typeorm": "0.3.16"
+    "rxjs": "7.8.1",
+    "typeorm": "0.3.17"
   },
   "devDependencies": {
     "@faker-js/faker": "8.0.2",
-    "@nestjs/cli": "9.0.0",
-    "@nestjs/schematics": "9.0.0",
-    "@nestjs/testing": "9.0.0",
+    "@nestjs/cli": "10.1.7",
+    "@nestjs/schematics": "10.0.1",
+    "@nestjs/testing": "10.0.5",
     "@types/bcrypt": "5.0.0",
-    "@types/express": "4.17.13",
-    "@types/jest": "29.5.1",
-    "@types/node": "18.16.12",
+    "@types/express": "4.17.17",
+    "@types/jest": "29.5.2",
+    "@types/node": "20.4.0",
```

Comment:
`@types/node` dependency has versions recommended for particular node versions. As we are using current LTS which is Node 18, we should require a `@types/node` version dedicated for 18. The latest one available according to [this page](https://www.npmjs.com/package/@types/node?activeTab=versions) is `18.16.19`. `20.4.0` (and newer) should be used with Node 20 (it will become LTS on 24.10.2023).